plugins/modules/ipahbacsvcgroup.py

```diff
@@ -98,7 +98,8 @@
 """
 
 from ansible.module_utils.ansible_freeipa_module import \
-    IPAAnsibleModule, compare_args_ipa, gen_add_del_lists
+    IPAAnsibleModule, compare_args_ipa, gen_add_del_lists, gen_add_list, \
```

Comment:
Is `ipahbacsvcgroup` in the PR by accident?

Reply:
fixed